components/HeroBlockVariation.vue

```diff
@@ -0,0 +1,54 @@
+<template>
```

Comment:
How does this differ from the `HeroBlock` component? We might want to think about naming this something different, as the `Hero` on the page is usually the first section, so if we're using this elsewhere, it wouldn't be a `Hero`.

Reply:
The `HeroBlock` component generally is ~600px and the text content next to it is 450-500px

I renamed `HeroBlockVariation` - the main/maybe only difference is that it has 1:1 ratio between image and text - something generic like `DefaultSection` work?